src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
 
 /**
  * @module TextInput
```

Comment:
While we are in here, would be great to make this a `PureComponent` vs `Component` as it just manages a simple state